spec/lib/ht/search_client/remote_spec.rb

```diff
@@ -55,6 +55,36 @@ def endpoint
         end
       end
 
+      context 'with unexpected params' do
+        let(:params) { { not_a_param: 'bad value' } }
+        let(:request_url) { 'http://username:password@test.com/foo?per_page=32&page=1&currency=EUR&order=sqs_score' }
+
+        it 'should add the parameters to the query' do
+          subject.perform
+          expect(stub_request :get, request_url).to have_been_requested
+        end
```

Comment:
I am struggling to understand this test

`should add the parameters to the query` => `expect(stub_request :get, request_url).to have_been_requested`

O_o could you explain it to me?